x-pack/plugins/security_solution/public/cases/attachments/external_reference_children.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { EuiMarkdownFormat } from '@elastic/eui';
+import type { IExternalReferenceMetaDataProps } from './types';
+
+export const ContentWrapper = styled.div`
+  padding: ${({ theme }) => `${theme.eui.euiSizeM} ${theme.eui.euiSizeL}`};
+  text-overflow: ellipsis;
+  word-break: break-word;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+`;
+
+const AttachmentContent = (props: {
+  externalReferenceMetadata: {
+    comment: IExternalReferenceMetaDataProps['externalReferenceMetadata']['comment'];
+  };
+}) => {
+  const {
+    externalReferenceMetadata: { comment },
+  } = props;
+
+  return comment.trim().length > 0 ? (
+    <ContentWrapper>
+      <EuiMarkdownFormat grow={true}>{comment}</EuiMarkdownFormat>
```

Comment:
I'm assuming this is the behaviour today (all comments are assumed to be `markdown`)?

If not, and this is a new change - Just keep in mind that because it treating it as `markdown`, the value displayed to the user might not match the value we display to them in other areas of the UI (ex. response actions log history). Could that be confusing?

Reply:
Your assumption is correct :) the previous implementation in Cases also use EuiMarkdownFormat with the ContentWrapper. There are some more configuration in place but I don't think they are needed in our use case. 
I am not expecting the behavior to be different, but if you could think of a specific use case I would be happy to reproduce it and verify if we're good.

Reply:
It it was using this before, then the content of the message might have already been different from what is seen in the Response action log. I'm ok with leaving it as is here. Thanks for the response.